pkg/jobs/scheduled_job.go

```diff
@@ -256,15 +247,15 @@ func (j *ScheduledJob) ScheduleExpr() string {
 func (j *ScheduledJob) Pause(reason string) {
 	j.rec.NextRun = time.Time{}
 	j.markDirty("next_run")
-	j.AddScheduleChangeReason(reason)
+	j.SetScheduleStatus(reason)
 }
 
 // Unpause resumes running this schedule.
 func (j *ScheduledJob) Unpause(reason string) error {
 	if err := j.SetSchedule(j.rec.ScheduleExpr); err != nil {
 		return err
 	}
-	j.AddScheduleChangeReason(reason)
+	j.SetScheduleStatus(reason)
```

Comment:
I'm not convinced we want Unpause setting a status, or pause either. IMO i'd remove `reason` from both of these, and during inc schedule creation just call Pause + SetScheduleStatus directly.